### PR TITLE
completes 5-patch-votes

### PR DESCRIPTION
**Pull request by RobertCox972**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -71,5 +71,42 @@ describe('GET article by id', () => {
 
 
         })
+        describe('PATCH inc_votes by ID', () => {
+            describe('PATCH inc_votes by ID', () => {
```
Comment (Poonam-raj) [inline __tests__/app.test.js:75]: Unnecessary describe block within a describe block I think

File: __tests__/app.test.js [hunk 2]
```diff
@@ -71,5 +71,42 @@ describe('GET article by id', () => {
...
+                    .patch('/api/articles/test')
+                    .expect(400)
+                    .then(({ body }) => {
+                        expect(body.msg).toBe('Bad Request');
+                    })
+                })
+                    test('returns status 400 when given invalid vote increment ', () => {
+                        return request(app)
+                        .patch('/api/articles/1')
+                        .send({inc_votes : test})
```
Comment (Poonam-raj) [inline __tests__/app.test.js:93]: test should be in a string to avoid a false fail from an undefined 'test' variable

File: __tests__/app.test.js [hunk 3]
```diff
@@ -71,5 +71,42 @@ describe('GET article by id', () => {
...
+                        .then(({ body }) => {
+                            expect(body.msg).toBe('Bad Request');
+                        })
+                   })
+            test('Increments the vote by the given inc_votes ', () => {
+                return request(app)
+                .patch('/api/articles/1')
+                .send({inc_votes : 18})
+                .expect(201)
+                .then(({ body: {articles, article}}) => {
```
Comment (Poonam-raj) [inline __tests__/app.test.js:104]: articles here isn't relevant, there shouldn't be a articles key to destructure

File: __tests__/app.test.js [hunk 4]
```diff
@@ -71,5 +71,42 @@ describe('GET article by id', () => {
...
+                    expect(article).toBeInstanceOf(Array)
```
Comment (Poonam-raj) [inline __tests__/app.test.js:105]: you should be sending back a object of one article on the key of 'article' so this expect line on 105 should be expecting an instance of an object not array

**Review by Poonam-raj — APPROVED** [review]
Few edits needed in your testing including the structure of what you're sending back but otherwise looking good - address the comments and feel free to merge :D

File: __tests__/app.test.js (hunk 3 above)
Comment (RobertCox972) [reply __tests__/app.test.js:104]: That was left over from when I was trying to get the pre-incremented value back to compare. Thank you!

**Pull request merged**